pom.xml

```diff
@@ -311,6 +311,12 @@
             <version>${spring.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.retry</groupId>
+            <artifactId>spring-retry</artifactId>
+            <version>1.2.4.RELEASE</version>
+            <optional>true</optional>
```

Comment:
should we somewhere document, one needs to add `spring-retry` when configuring retries?

Reply:
I would also add comment here, that you need this dependency when using `RetrySettings` ;)

Reply:
Added.